packages/api-bindings/src/graphql/comments.graphql

```diff
@@ -12,9 +12,16 @@ query Comments(
   $limit: LimitScalar!
   $cursor: Cursor
   $sources: [Sources!]
+  $metadata: PublicationMetadataFilters
```

Comment:
should we update our field policies to properly set key args? Right now it's will not keep different `metadata` queries as separate items in the apollo cache.